CI failed

Clippy in CI reports:

```
error: redundant redefinition of a binding
  --> src/components/issue_asset_page.rs:50:9
   |
50 |     let presision = 0;
   |         ^^^^^^^^^
...
85 |             let presision = presision;
   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = help: remove the redefinition of `presision`
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#redundant_locals
   = note: `#[deny(clippy::redundant_locals)]` on by default

error: could not compile `shiro-frontend` (bin "shiro-frontend") due to previous error
Error: Process completed with exit code 101.
```

--- a/src/components/issue_asset_page.rs
+++ b/src/components/issue_asset_page.rs
@@ -82,7 +82,6 @@ pub fn page(_props: &IssueAssetPageProps) -> Html {
 
             let ticker = ticker.clone();
             let name = name.clone();
-            let presision = presision;
             let amounts = amounts.clone().iter().map(|s| s.to_string()).collect();
 
             let asset = IssueParams {

